lib/engine/game/g_1880/game.rb

```diff
@@ -304,6 +304,7 @@ def event_token_cost_doubled!
         def setup
           @full_cap_event = false
           @foreign_investors_can_operate = true
+          @par_order = { 100 => [], 90 => [], 80 => [], 70 => [] }
```

Comment:
I have a feeling that you'll have to populate this with nils, so someone can pick a specific spot in the order.

Reply:
The player is allowed to pick any of the 4 spots within each par value